lib/html.js

```diff
@@ -65,10 +64,6 @@ module.exports = function(el, compute, viewInsertSymbolOptions) {
 	}
 	//!steal-remove-end
 
-	var doNotUpdateRange = function() {
```

Comment:
This part of the patch set is boyscouting from the queue changes I made recently.  Since updateRange is never queued, it is not necessary to remove it from the dom queue.